packages/now-next/src/utils.ts

```diff
@@ -302,6 +302,14 @@ export type Redirect = Rewrite & {
   statusCode?: number;
 };
 
+export type Header = {
```

Comment:
This type (and many others above) are available in routing utils:

```js
import { NowHeader } from `@now/routing-utils`;
```

Reply:
Updated to re-use some types from `@now/routing-utils`